### DISCUSSION TITLE
Feature: build docker image

### DISCUSSION THREAD

**Discussion by davidclaeysquinones** [iptv-org/community#1092] [Show and tell]
I think that having a Docker image of the program available would simplify deployment a lot.
This [repository](https://github.com/davidclaeysquinones/epg-info-docker) is an attempt to do exactly this.
Feel free to incorporate my work into the project or to provide any suggestion.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/awesome-iptv/pull/284
